nltk/tokenize/stanford_segmenter.py

```diff
@@ -64,13 +65,20 @@ def __init__(self,
                 env_vars=('STANFORD_SEGMENTER',),
                 searchpath=(), url=_stanford_url,
                 verbose=verbose)
-        slf4j = find_jar(
+        if path_to_slf4j is not None:
+            if path_to_slf4j == 'PLACEHOLDER':
```

Comment:
The 'PLACEHOLDER' is to enable the old version to execute.
When using old version, and write

segmenter = StanfordSegmenter()

We need to find the correct slf4j-api.jar, and the find_jar_iter in internals.py would execute the following:

    if path_to_jar is not None:
        if os.path.isfile(path_to_jar):
            yielded = True
            yield path_to_jar
        else:
            raise LookupError('Could not find %s jar file at %s' %
                            (name_pattern, path_to_jar))

and ignore the environment variables. We can only find correct slf4j-api.jar using the environment variables. So in this default situation, we need to set path_to_jar to None.